azerothcore/azerothcore-wotlk
# fix(Core/Item): Implement mount speed mod items

## Changes Proposed:
-  Implement Riding Crop / Skybreaker Whip
-  Group these items together in one script
- Riding Crop / Skybreaker Whip SHOULD NOT stack with any other item
- Carrot on a stick + Mithril Spurs SHOULD stack

## Tests Performed:
- Builds
- Tested in-game
![image](https://user-images.githubusercontent.com/24550914/104222646-e0bd2900-5442-11eb-832d-b7d64fa9c33f.png)
![image](https://user-images.githubusercontent.com/24550914/104222674-ea469100-5442-11eb-9d4a-28f6506ea49f.png)
![image](https://user-images.githubusercontent.com/24550914/104222693-f16d9f00-5442-11eb-8d04-7567d0d2924f.png)
![image](https://user-images.githubusercontent.com/24550914/104222722-f894ad00-5442-11eb-85c9-53bea0e6c594.png)




## How to Test the Changes:

Make sure to be <= lvl 70
1. .learn 61425 (Tundra Mammoth alliance)
2. .add 32863/25653/11122/7969



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
